Gems/ROS2/Code/Source/Spawner/ROS2SpawnerComponent.cpp

```diff
@@ -169,19 +180,45 @@ namespace ROS2
 
         AZ::Transform transform;
 
-        if (auto spawnPoints = GetSpawnPoints(); spawnPoints.contains(spawnPointName))
+        if (isWGS)
         {
-            transform = spawnPoints.at(spawnPointName).pose;
+            ROS2::WGS::WGS84Coordinate coordinate;
+            AZ::Vector3 coordinateInLevel = AZ::Vector3(-1);
+            AZ::Quaternion rotationInENU = AZ::Quaternion::CreateZero();
```

Comment:
better be safe than sorry :
```suggestion
            AZ::Quaternion rotationInENU = AZ::Quaternion::CreateIdentity();
```